test/unit/src/constants.tests.js

```diff
@@ -70,7 +70,7 @@ export default QUnit.module( 'Constants', () => {
 		assert.equal( Constants.ReinhardToneMapping, 2, 'ReinhardToneMapping is equal to 2' );
 		assert.equal( Constants.CineonToneMapping, 3, 'CineonToneMapping is equal to 3' );
 		assert.equal( Constants.ACESFilmicToneMapping, 4, 'ACESFilmicToneMapping is equal to 4' );
-		assert.equal( Constants.CustomToneMapping, 5, 'CustomToneMapping is equal to 5' );
+		assert.equal( Constants.CustomToneMapping, 6, 'CustomToneMapping is equal to 6' );
```

Comment:
I don't fully understand this test file. What's the value of testing all the constant values?

Reply:
I think it's nice to test _some_ of the enum values, e.g. to verify they're still exported ... but yeah this is probably a little excessive.

Reply:
The idea is to ensure the values behind a constant (`6` in the case of `CustomToneMapping`) does not change. This is important when scene are serialized and need to be restored.

Because of that, the value of `CustomToneMapping` should not be changed. Give `AgXToneMapping` the value of `6`.

Reply:
Gotcha - just reverted the change.

Reply:
To clarify - which scene serialization are we talking about? User serialization? Or the `toJSON` format?

Reply:
For example the editor saves the selected tone mapping type. Users could do the same in their application.